content/en/synthetics/settings/_index.md

```diff
@@ -66,6 +69,24 @@ Note: Variable values are updated whenever the test they are extracted from runs
 [3]: https://restfulapi.net/json-jsonpath/
 {{% /tab %}}
 
+{{% tab "MFA Token" %}}  
+ 
+In the **Global Variables** tab of your **Settings** page, click **Create Global Variable** and enter a secret key or upload a QR code from your authentication provider:
+
+1. In **Choose variable type**, select **MFA Token**.
+2. In **Define Variable**, enter a **Variable Name**. Your variable name can only use uppercase letters, numbers, and underscores.
+3. Enter a **Description** for your variable (optional).
+4. Select **Tags** to associate with your variable (optional).
+5. Enter the **Secret Key** to your variable or upload a QR code image.
+6. Click **+ Generate** to create an OTP. You can copy the generated OTP with the **Copy** icon.
+7. In **Permissions settings**, restrict access to your variable based on roles in your org. For more information about roles, see the [RBAC documentation][1].
```

Comment:
Content of 7. leaves in [this section][https://docs.datadoghq.com/synthetics/settings/?tab=specifyvalue#restrict-access], so should be removed.